src/options/Main/OptionsList/index.jsx

```diff
@@ -13,7 +14,8 @@ export default class OptionsList extends Component {
       isLoading: true,
       mode: 'tab',
       showEmptySearchSuggestions: true,
-      enableFuzzySearch: true
+      enableFuzzySearch: true,
+      theme: 'light'
```

Comment:
Instead of using strings can we use something similar to enums? I think it would be sufficient to create an object that maps to the valid themes. Then we could just access them using `Themes.Light`. This will help prevent mistakes with typos 😉. I realize the modes don't do this but that should probably be refactored later.

Reply:
ok, I'll have a look into it (: